cloudify_ansible/utils.py

```diff
@@ -389,3 +411,14 @@ def cleanup(ctx):
     instance = _get_instance(ctx)
     for key, _ in instance.runtime_properties.items():
         del instance.runtime_properties[key]
+
+
+def set_playbook_config_as_runtime_properties(_ctx, config):
+    """
+    Set all playbook node instance configuration as runtime properties
+    :param _ctx: Cloudify node instance which is instance of CloudifyContext
+    :param config: Playbook node configurations
+    """
+    if config and isinstance(config, dict):
+        for key, value in config.items():
+            _ctx.instance.runtime_properties[key] = value
```

Comment:
shouldn't we use a `config` runtime property to store the config in? What if there are other keys in there that override keys of other runtime properties?

Reply:
This is only used for `PlayBook` Node and this will run as part of configure operation for `PlayBook` so that it can later on used when run the playbook to the host